### ISSUE TITLE
Rename `derive_derived_param()`

### ISSUE THREAD

**Issue by thomas-neitmann** [pharmaverse/admiral#1229]
Given our naming convention this function should be called `derive_param_derived()`.

**Comment by thomas-neitmann** [comment]
Agree the current name isn't all that telling... I'm struggling with finding something better myself, though. Any thoughts @pharmaverse/admiral?

**Comment by gg106046** [comment]
How about `derive_param_computed()` ? Since we're computing the new parameter using the analysis value of other parameters.

**Comment by thomas-neitmann** [comment]
@PuzzledFace Can you open a pull request such that I can take a look at your changes and see if I have a solution for #1252?

**Comment by PuzzledFace** [comment]
Thomas,

PuzzledFace is me, in case you hadn’t worked it out!

Trying to open a pull request, but I’m having  PAT issues.  bear with me…

Cheers,

John


John Kirkpatrick

Tile Barn
High Haden Road
Glatton PE28 5RU

T: 01487 830377
E: ***@***.***

**Comment by bms63** [comment]
Hi @PuzzledFace We leave issues open until a release is pushed out. If you PR title has close #1229 in it then it will auto-close the issue once devel is merged into main
